Enterprise-CMCS/managed-care-review
# MCR-4456: update submission date to et time zone

## Summary
[MCR-4456](https://jiraent.cms.gov/browse/MCR-4456)

- Update GraphQL schema
   - Update `Contract` and `Rate` types field `initiallySubmittedAt` to be `DateTime` so we can convert the UTC timestamp to ET.
- Update display dates to be ET timezone.
   - Submitted dates:
      - Summary pages
   - Document added dates:
      - Summary pages
      - Q&A page
      - Linked rates on the Rate details page
   - Email templates
   - Dashboard pages
- Update display dates format to `MM/DD/YYYY`
   - Banners
      - Submission Unlocked
         <img src="https://github.com/user-attachments/assets/4a380be7-d853-4bf0-81eb-b07b0c567b60" width="300" />
      - Submission Updated
         <img src="https://github.com/user-attachments/assets/c2bade10-1552-4d5d-9583-ecd835697c3b" width="300" />
      - Rate withdrawn
         <img src="https://github.com/user-attachments/assets/60e34c9b-0f4f-448d-a860-e1c18c661834" width="300" />
   - Change history
      <img src="https://github.com/user-attachments/assets/236ccca0-20bb-4c08-b922-5dd287933a15" width="450" />
   - Previous submission version date time text
      <img src="https://github.com/user-attachments/assets/146f8249-8f3f-43e1-97fb-97066e83e633" width="450" />
- Fixed Linked rate document date added not showing up

#### Related issues

#### Screenshots

#### Test cases covered

- Updated a whole slew of tests that verifies UTC is converted to ET.



## QA guidance

- The Unit tests should validate that the conversions work since `vitest` and `jest` uses UTC time when you call `new Date()`. You wanted to test this in the app, make a submission at 10:00 PM ET, in UTC this would 1 day ahead of whatever date it was at 10 PM ET.
